foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/TestVertxUtils.java

```diff
@@ -51,6 +64,17 @@ public void testGetOrCreateVertx() throws InterruptedException {
     VertxUtils.closeVertxByName("ut");
   }
 
+  @Test
+  public void testCreateVertxWithFileCPResolving() throws IOException {
```

Comment:
1.force delete dir
2.test with create dir
3.force delete dir
4.test with not create dir

Reply:
already fixed